CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+> ![Important]
+> This version makes the new view renderer used by Session Replay generally available.
```

Comment:
`m`: I think it is essential to briefly mention that the V2 renderer is better than V1. Something like this could work.
```suggestion
> This version enables the better view renderer V2 used by Session Replay generally available and enables it by default.
```

Reply:
Happy to iterate, is the updated version okay?